src/test/java/edu/ksu/canvas/attendance/config/TestSpringMVCConfig.java

```diff
@@ -84,6 +87,15 @@ public LtiLaunch ltiLaunch() {
     public AttendanceAssignmentRepository attendanceAssignmentRepository () { return Mockito.mock(AttendanceAssignmentRepository.class); }
 
     @Bean
+    public AssignmentSubmitter assignmentSubmitter() { return Mockito.mock(AssignmentSubmitter.class); }
```

Comment:
Just a note that Spring will still try to inject the members even though this is a mock. This will require you to create beans for all of those members and all of the members that that bean depends on and so on. This means ther will be a lot of unused beans declared in the test context. 

What most people would expect is that the members would not have to be injected because the members are never used since it is a mock.